versioned_docs/version-v2.18.x/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,677 @@
+{
+  ApplicationControlled Off
+  CertificateLabel # Leave empty to avoid sending a client certificate (i.e. if the keyring has a default certificate)
+  SecondaryMap Off
+}
+```
+
+:::important
+
+- Outbound connections from the Gateway to southbound services (onboarded services) must not send the server certificate if the service accepts X.509 Client Certificate authentication. If the server certificate is sent, it is the server user who would be authenticated.
+
+:::
+
+#### Services that validate tokens against the API Mediation Layer
+
+In this scenario, the services will issue a request against the API Gateway to validate the received authentication token.
+
+This includes services that set `zoweJwt` as authentication scheme, those that require an Open ID Connect (OIDC) token or forwarded X.509 certificates.
+
+In this case it's necessary to have an Outbound rule from the service to the API Gateway.
+
+These service will also already have an outbound rule set for the onboarding process against the Discovery Service.
+
+Ensure these rules are followed:
+
+- Outbound rule to Discovery Service: Sends X.509 Client Certificate to authorize the onboarding.
+- Outbound rule to API Gateway: __Do not__ set a Client Certificate.
+
+### Ciphers
+
+:::note
+This list of ciphers is provided as an example only. Actual ciphers should be customized according to your specific configuration.
+:::
+
+The list of supported ciphers should be constructed according to the TLS supported versions.
+Ensure that the cipher list has matches with non-AT-TLS-aware clients.
+
+<details>
+<summary>Click here for an example of Cipher parameters.</summary>
+
+```bash
+TTLSCipherParms CipherParms
+{
+  V2CipherSuites TLS_RC4_128_WITH_MD5
+  V2CipherSuites TLS_RC4_128_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_DES_192_EDE3_CBC_WITH_MD5
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_AES_128_GCM_SHA256
+  V3CipherSuites TLS_AES_256_GCM_SHA384
+  V3CipherSuites TLS_CHACHA20_POLY1305_SHA256
+}
 ```
 
-## Component-Specific Configuration
+</details>
+
+## Using AT-TLS for API ML in High Availability
+
+AT-TLS settings for a Zowe installation configured in High Availability mode do not differ extensively. Changes need to be made to the previously described rules to allow for cross-lpar communication:
+
+Ensure that the `RemoteAddr` setting in the rules accounts for the following connections:
+
+- Discovery Service to Discovery Service. This is the replica request.
+- Gateway Service to southbound services (including app-server and ZSS) running in another LPAR.
```

Comment:
maybe instead of `running in another LPAR` put a not, all rule should be both inside the LPAR and against other LPARs (with a running service)